apps/marketplace/components/Reports/ReportItem.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import includes from 'lodash/includes'
+import { uniqueID } from '../shared/utils/helpers'
+import * as styles from './ReportItem.scss'
+
+const computeClassname = (base, alt, formatted) =>
+  cx(base, {
+    [alt]: formatted
+  })
+
+const imageRoot = '/static/media/'
+
+const ReportItem = props => {
+  const { heading, subitems } = props
+  const mobile = true
+
+  return (
+    <div className="col-sm-12 report-item">
+      <div className={styles.reportItem}>
+        {heading &&
+          <h2 className={`${styles.reportItemHeading} uikit-display-2`}>
+            {heading}
+          </h2>}
+        {subitems.map((subitem, id = uniqueID()) =>
+          <p className={`${styles.reportSubitemContainer} ${subitem.fullWidth ? 'col-sm-12' : 'col-sm-6'}`} key={id}>
+            {subitem.text &&
+              <span className={styles.reportItemText}>
+                {subitem.text}
+              </span>}
+            {subitem.image &&
+              <span
+                className={computeClassname(
+                  styles.reportItemImage,
+                  styles.reportFormattedQuoteImage,
+                  subitem.formattedImage
+                )}
+              >
+                {subitem.imageCaption
+                  ? <figure
+                      tabIndex="0"
+                      style={{ margin: 0 }}
+                      aria-describedby={`${subitem.image.replace('768-', '').replace('.svg', '')}-figcaption`}
+                    >
+                      <img
+                        className={
+                          includes(
+                            [
+                              'briefs-by-phase',
+                              'number-of-sellers-per-area-of-expertise',
+                              'seller-applications-per-brief'
+                            ],
+                            subitem.image.replace('768-', '').replace('.svg', '')
+                          )
+                            ? styles.bigReportImage
+                            : styles.reportImage
+                        }
+                        src={`${imageRoot}${mobile ? subitem.mobileImage : subitem.image}`}
+                        alt={subitem.imageAlt}
+                      />
+                      <figcaption
+                        className={styles.sr_only}
+                        id={`${subitem.image.replace('768-', '').replace('.svg', '')}-figcaption`}
+                      >
+                        {subitem.imageCaption}
+                      </figcaption>
+                    </figure>
+                  : <img
```

Comment:
We should avoid using ternarie, it is better to just render a section above the render(). You can do something {renderThisSection()} and then on top you can just use if statements to return blocks.